code/modules/mob/living/simple_animal/slime/slime.dm

```diff
@@ -494,66 +369,78 @@
 		step_away(src,user,15)
 
-/mob/living/simple_animal/slime/get_mob_buckling_height(mob/seat)
-	if(..())
-		return 3
+///Spawns a crossed slimecore item
+/mob/living/simple_animal/slime/proc/spawn_corecross()
+	var/static/list/crossbreeds = subtypesof(/obj/item/slimecross)
+	visible_message(span_danger("[src] shudders, its mutated core consuming the rest of its body!"))
+	playsound(src, 'sound/magic/smoke.ogg', 50, TRUE)
+	var/selected_crossbreed_path
+	for(var/crossbreed_path in crossbreeds)
+		var/obj/item/slimecross/cross_item = crossbreed_path
+		if(initial(cross_item.colour) == slime_type.colour && initial(cross_item.effect) == crossbreed_modification)
+			selected_crossbreed_path = cross_item
+			break
+	if(selected_crossbreed_path)
+		new selected_crossbreed_path(loc)
+	else
+		visible_message(span_warning("The mutated core shudders, and collapses into a puddle, unable to maintain its form."))
+	qdel(src)
 
-///Sets the slime's current attack target
-/mob/living/simple_animal/slime/proc/set_target(new_target)
-	var/old_target = Target
-	Target = new_target
-	if(old_target && !SLIME_CARES_ABOUT(old_target))
-		UnregisterSignal(old_target, COMSIG_QDELETING)
-	if(Target)
-		RegisterSignal(Target, COMSIG_QDELETING, PROC_REF(clear_memories_of), override = TRUE)
-
-///Sets the person the slime is following around
-/mob/living/simple_animal/slime/proc/set_leader(new_leader)
-	var/old_leader = Leader
-	Leader = new_leader
-	if(old_leader && !SLIME_CARES_ABOUT(old_leader))
-		UnregisterSignal(old_leader, COMSIG_QDELETING)
-	if(Leader)
-		RegisterSignal(Leader, COMSIG_QDELETING, PROC_REF(clear_memories_of), override = TRUE)
-
-///Alters the friendship value of the target
-/mob/living/simple_animal/slime/proc/add_friendship(new_friend, amount = 1)
-	if(!Friends[new_friend])
-		Friends[new_friend] = 0
-	Friends[new_friend] += amount
-	if(new_friend)
-		RegisterSignal(new_friend, COMSIG_QDELETING, PROC_REF(clear_memories_of), override = TRUE)
-
-///Sets the friendship value of the target
-/mob/living/simple_animal/slime/proc/set_friendship(new_friend, amount = 1)
-	Friends[new_friend] = amount
-	if(new_friend)
-		RegisterSignal(new_friend, COMSIG_QDELETING, PROC_REF(clear_memories_of), override = TRUE)
-
-///Removes someone from the friendlist
-/mob/living/simple_animal/slime/proc/remove_friend(friend)
-	Friends -= friend
-	if(friend && !SLIME_CARES_ABOUT(friend))
-		UnregisterSignal(friend, COMSIG_QDELETING)
-
-///Adds someone to the friend list
-/mob/living/simple_animal/slime/proc/set_friends(new_buds)
-	clear_friends()
-	for(var/mob/friend as anything in new_buds)
-		set_friendship(friend, new_buds[friend])
+///Handles slime attacking restrictions, and any extra effects that would trigger
+/mob/living/simple_animal/slime/proc/slime_pre_attack(mob/living/simple_animal/slime/our_slime, atom/target, proximity, modifiers)
+	SIGNAL_HANDLER
+	if(isAI(target)) //The aI is not tasty!
+		to_chat(our_slime, "This being is incompatible, and not tasty.")
```

Comment:
```suggestion
		to_chat(our_slime, span_warning("This being is incompatible, and not tasty."))
```
or
```suggestion
		target.balloon_alert(our_slime, "not tasty!")
```

Reply:
I went with the balloon alert in the end. Tempting to change all the alerts in the slime files into balloons.